src/Serialization/main.jl

```diff
@@ -1,9 +1,135 @@
+################################################################################
+# Description of the saving and loading mechanisms
+#
+# We require that any types serialized through OSCAR are registered using the
+# @register_serialization_type macro, like so
```

Comment:
I think this should say
```suggestion
# @register_serialization_type macro. For more information, see its docstring.
```
And then move the explanation below into the docstring.